### PR TITLE
gitignore dashboard war overlays folder

### PR DESCRIPTION
**Pull request by sparkoo**
### What does this PR do?
updates gitignore with dashboard overlays folder

### What issues does this PR fix or reference?
n/a

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -65,6 +65,7 @@ Thumbs.db
 */overlays
 *~
 .directory
+assembly/assembly-dashboard-war/overlays/
```
Comment (mshaposhnik) [inline .gitignore]: Maybe just modify L65 to `**/overlays` ?

Comment (sparkoo) [reply .gitignore]: I was thinking about that, but it felt too wide for me. Do you think it won't cause issues when we would need to create `overlays` folder anywhere? (probably not and my worries are irrelevant here)

Comment (mshaposhnik) [reply .gitignore]: dont think it's a big issue... btw we can do `assembly/**/overlays` to narrower scope

Comment (sparkoo) [reply .gitignore]: updated

**Review by skabashnyuk — APPROVED** [review]

**Comment by sparkoo** [comment]
crw-ci-test

**Pull request merged**